src/views/TodosList.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from "react";
+import { todos } from "../fakeData";
+
+export default class TodoList extends Component {
+    render() {
+        return (
+            <React.Fragment>
+                <h3 className="ml-3 mt-3">All Todos</h3>
+                <div className="row">
+                    <div className="col-md-12">
+                        {
+                            todos.map((item, index) => (
```

Comment:
Create function for manipulation and return. then just bind that function here.
e.g.
`{renderTodoList()}`

Reply:
Okay